### PR TITLE
Adding tlg1181.tlg001 from Schiefsky's file to FirstIK-grc

### PR DESCRIPTION
**Pull request by rebeccadraughon**
Changed tlg1181.tlg001 from Schiefsky's list to DDD compliance; added to First1K-grc.

### CODE REVIEW

File: data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,783 @@
...
+      <ref target="https://archive.org/details/aristarchusofsam00heatuoft"/>
+     </biblStruct>
+    </listBibl>
+   </sourceDesc>
+  </fileDesc>
+  <encodingDesc>
+   <p>Text encoded in accordance with the latest EpiDoc standards</p>
+   <refsDecl n="CTS">
+    <cRefPattern matchPattern="(.+).(.+)" n="section"
+     replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])"/>
```
Comment (sonofmun) [inline data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml]: There is no second level of the citation scheme here, only "chapter"s. So the "section" cRefPattern should be deleted.

File: data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,783 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+ <teiHeader xml:lang="eng">
+  <fileDesc>
+   <titleStmt>
+    <title>Aristarchus of Samos, the Ancient Copernicus: a History of Greek Astronomy to
```
Comment (sonofmun) [inline data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml]: The `<title>` element in the `<titleStmt>` should be the title of the work, not the title of the volume. According to the TLG, the title of this work is "De magnitudinibus et distantiis solis et lunae". Make sure also to put an xml:lang="lat" attribute on the `<title>` element.

File: data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml [hunk 3]
```diff
@@ -0,0 +1,783 @@
...
+     <resp>Lead Developer (University of Leipzig) 2015 - 2017</resp>
+    </respStmt>
+    <respStmt>
+     <persName>Bruce Robertson</persName>
+     <resp>Technical Advisor (Mount Allison University)</resp>
+    </respStmt>
+   </titleStmt>
+   <publicationStmt>
+    <authority>Harvard College Library</authority>
+    <idno type="tlg1181.tlg001.1st1K-grc.xml"/>
```
Comment (sonofmun) [inline data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml]: The `type` attribute here should be "filename" and then the filename itself should be in the text of the element, i.e., `<idno type="filename">tlg1181.tlg001.1st1K-grc1.xml</idno>`.

File: data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml [hunk 4]
```diff
@@ -0,0 +1,783 @@
...
+      <monogr>
+       <title>Aristarchus of Samos, the Ancient Copernicus: a History of Greek Astronomy to
+        Aristarchus, Together with Aristarchus's Treatise on the Sizes and Distances of the Sun and
+        Moon</title>
+       <editor>
+        <persName>
+         <name>Sir Thomas Heath</name>
+        </persName>
+       </editor>
+       <author ref="urn:cts:greekLit:tlg1181.tlg001">Aristarchus</author>
```
Comment (sonofmun) [inline data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml]: The URN for the author should not include the "tlg001". It should only be "urn:cts:greekLit:tlg1181".

File: data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml [hunk 5]
```diff
@@ -0,0 +1,783 @@
...
+       <imprint>
+        <publisher>Oxford</publisher>
+        <pubPlace>The Claredon Press</pubPlace>
+        <date>1913</date>
+       </imprint>
+       <biblScope unit="volume"/>
+      </monogr>
+      <ref target="https://archive.org/details/aristarchusofsam00heatuoft"/>
```
Comment (sonofmun) [inline data/tlg1181/tlg1181.tlg001.1st1K-grc1.xml]: The `<ref>` element should contain a text naming where this came from, i.e., `<ref target="https://archive.org/details/aristarchusofsam00heatuoft">The Internet Archive</ref>`

**Comment by sonofmun** [comment]
There are some changes that need to be made besides those in the XML file.

1. There needs to be a sub-folder for the work. So the TEI-XML file should be in /data/tlg1181/tlg001
2. In the author folder (i.e., /data/tlg1181) there needs to be a `__cts__.xml` file with the metadata for the author. Check, e.g., https://github.com/OpenGreekAndLatin/First1KGreek/blob/master/data/tlg0015/__cts__.xml for an example.
3. In the work-level folder (i.e., /data/tlg1181/tlg001), there also needs to be a `__cts__.xml` metadata file with the information about the work and each of the editions. See https://github.com/OpenGreekAndLatin/First1KGreek/blob/master/data/tlg0015/tlg001/__cts__.xml for an example.

**Comment by lcerrato** [comment]
It looks like tlg1181.tlg001 was added at a later date. I have finished the updates to the other file here tlg0087.tlg035 in this PR https://github.com/OpenGreekAndLatin/First1KGreek/pull/2822

**Pull request closed without merging**